src/funcs.c

```diff
@@ -2948,16 +2948,16 @@ void open_current_dir()
 		msg_box((const char*)GetMBString(MSG_SelectGameFromList));
 		return;
 	}
-	
+
```

Comment:
I think this and line 2958 are due to the .editorconfig file kicking in. They weren't in the diff inside VS Code.